falco.yaml

```diff
@@ -166,4 +166,15 @@ program_output:
 
 http_output:
   enabled: false
-  url: http://some.url
\ No newline at end of file
+  url: http://some.url
+
+grpc:
+  enabled: false
+  bind_address: "0.0.0.0:5060"
+  threadiness: 8
+  private_key: "/etc/falco/certs/server.key"
+  cert_chain: "/etc/falco/certs/server.crt"
+  root_certs: "/etc/falco/certs/ca.crt"
+
+grpc_output:
```

Comment:
Was this a design choice? Why do we have `grpc_output` separate from `grpc`?

Reply:
Yes Kris.

They are separated because we want the Falco Outputs feature to be separated from future gRPC features. In order to be able to selectively enable one thing or the other.